### ISSUE TITLE
addon-a11y - Add ability to mark 'Incomplete' issues as complete

### ISSUE THREAD

**Issue by Abibubble** [storybookjs/storybook#18580]
**Is your feature request related to a problem? Please describe**
My team is creating a Storybook design system to share between developers and designers. With the current addon-a11y model, some things are showing as Incomplete issues. We've manually checked these and the issues we're finding are non-issues. However, it doesn't look fully professional having one or two 'Incomplete' issues showing for each component, when we've manually checked them and they're not issues.

**Describe the solution you'd like**
Ideally, a way of marking the 'Incomplete' issue as completed or checked would be fantastic. This would hopefully move the issue into 'Passed', or mark it as 'manually checked', to show the other developers and designers that we haven't just left accessibility issues in our components.

**Describe alternatives you've considered**
The other solution we considered was to remove the 'Incomplete' tab altogether, but it doesn't appear that there's a way to do that either (and that removes visibility for the issues).

**Are you able to assist to bring the feature to reality?**
I'm happy to help in any way that I can

**Additional context**
The image below shows the main issue we're having. Colour contrast is marked as Incomplete. We've checked it manually (we're literally only using black and white for most of our components), but there isn't a way to clear it from the Incomplete tab.

<img width="786" alt="Screenshot 2022-06-27 at 14 06 43" src="https://user-images.githubusercontent.com/71400099/175948760-0c478ba4-23e3-44fc-a35c-63a34cdc4c89.png">

**Comment by ChristianTumamo** [comment]
I think it might have something to do with inline javascript being ran. It can interfere with the addon. Not sure what the solution, got the same issue here.
